physical/gcs/gcs.go

```diff
@@ -61,15 +97,22 @@ func NewGCSBackend(conf map[string]string, logger log.Logger) (physical.Backend,
 		if err != nil {
 			return nil, errwrap.Wrapf("failed parsing max_parallel parameter: {{err}}", err)
 		}
-		if logger.IsDebug() {
-			logger.Debug("physical/gcs: max_parallel set", "max_parallel", maxParInt)
+		if logger.IsWarn() {
+			logger.Warn("physical/gcs: max_parallel set", "max_parallel", maxParInt)
 		}
 	}
 
+	haEnabled := os.Getenv("GCS_HA_ENABLED")
+	if haEnabled == "" {
+		haEnabled = conf["ha_enabled"]
+	}
+	haEnabledBool, _ := strconv.ParseBool(haEnabled)
```

Comment:
The rest of the backends have the same behaviour. If not set = false

Reply:
I meant that it currently doesn't seem like this defaults to "false" and will error out if neither config is provided.

Reply:
Ah, nvm. I forgot strconv.ParseBool will still return default false on error.

Reply:
Yea, but I don't like that we are silently discarding the error. If the user typo-ed "treu", it would silently fail to false. [Here's what I did](https://github.com/hashicorp/vault/blob/5b4abc09ddd82be99e05264a91b06bc18ab1447a/physical/spanner/spanner.go#L126-L138) in the Spanner backend